libvmi/arch/amd64.c

```diff
@@ -283,7 +296,7 @@ GSList* get_pages_ia32e(vmi_instance_t vmi, addr_t npt, page_mode_t npm, addr_t
             goto done;
 
         uint64_t pdpte_index;
-        for (pdpte_index = 0; pdpte_index < IA32E_ENTRIES_PER_PAGE; pdpte_index++, pdpte_location++) {
+        for (pdpte_index = 0; pdpte_index < IA32E_ENTRIES_PER_PAGE; pdpte_index++, pdpte_location += entry_size) {
```

Comment:
This seems to be an unrelated change, different bug?

Reply:
Yes, that's why I've created a separate commit for this change. I've discovered it while looking for the other bugs. Would you like me to create a separate PR for it?

Reply:
Not necessary, just checking